### ISSUE TITLE
Add guidance on decorative images in our automated message

### ISSUE THREAD

**Issue by kendallgassner** [github/accessibility-alt-text-bot#12]
### What

A decorative image should not have alt text and instead should have null alt text: `alt=''`. 

Currently we do not have guidance on decorative images in our automated message. Let's add some!

**Comment by khiga8** [comment]
@kendallgassner Since images on GitHub all render as a link, should we continue to require alt text for images to avoid links with no name, for now?

**Comment by kendallgassner** [comment]
Oh good thought 🤔! 

Maybe we can add that as a note instead.
